qa/rpc-tests/test_framework/test_framework.py

```diff
@@ -206,7 +206,7 @@ def main(self,argsOverride=None,bitcoinConfDict=None,wallets=None):
         logging.info("Random seed: %s" % self.randomseed)
 
         if self.options.tmpdir is None:
-            self.options.tmpdir = self.options.tmppfx
+            self.options.tmpdir = os.path.join(self.options.tmppfx, str(self.options.port_seed))
```

Comment:
why adding back seed port in the construction of `tmpdir` path?

Reply:
I mean this is not needed to restore the `tmppfx` cause it was never removed in the first place.

Reply:
fixed in d639deb